api/v2/views/image_version_boot_script.py

```diff
@@ -10,12 +10,12 @@ class VersionFilter(django_filters.FilterSet):
     version_id = django_filters.CharFilter(method='filter_by_uuid')
     created_by = django_filters.CharFilter(method='filter_owner')
 
-    def filter_owner(self, queryset, value):
+    def filter_owner(self, queryset, query_key, value):
```

Comment:
`query_key` is not used. Should it? Or is this laying groundwork/WIP? Same for a few places below.